libc-test/build.rs

```diff
@@ -2458,7 +2449,17 @@ fn test_linux(target: &str) {
                                            field == "_pad2" ||
                                            field == "ssi_syscall" ||
                                            field == "ssi_call_addr" ||
-                                           field == "ssi_arch"))
+                                           field == "ssi_arch")) ||
+        // FIXME: After musl 1.1.24, it have only one field `sched_priority`,
+        // while other fields become reserved.
```

Comment:
Can you elaborate on what this means?

Reply:
I placed the diff of musl in [the initial comment](https://github.com/rust-lang/libc/pull/1588#issue-339146716)

Reply:
Thanks, there are functions for mapping Rust field names to C field names in ctest, e.g., https://docs.rs/ctest/0.2.22/ctest/struct.TestGenerator.html#method.field_name . So maybe we can just translate the Rust names to the new __reservedX names for now ?

Reply:
@gnzlbg 
>  So maybe we can just translate the Rust names to the new __reservedX names for now ?

This does not work for `struct { time_t __reserved1; long __reserved2; } __reserved2[2];`.
The anonymous struct does not match `struct timespec`.  [Build log](https://travis-ci.org/oxalica/libc/jobs/614414350#L5198)

We still need to skip these fields at least.

Reply:
Yes, you are right.